bitsail-connectors/connector-assert/src/main/resources/bitsail-connector-unified-assert.json

```diff
@@ -0,0 +1,9 @@
+{
+  "name": "bitsail-connector-unified-assert",
```

Comment:
should we remove `unified` here?

Reply:
I noticed that connector V1 is uniformly named `bitsail-connector-unified-xxx`, like this, the same naming convention is used here.

![image](https://user-images.githubusercontent.com/42756849/209890790-7a77d1a4-6998-401c-b919-dcf9d626d1dd.png)

Reply:
ok I see